dev/post-install.js

```diff
@@ -0,0 +1,7 @@
+import {writeFileSync} from 'fs';
+
+writeFileSync('./node_modules/rsuite/lib/styles/normalize.less', '', (err) => {
```

Comment:
we need to check for existence before writing, and throw an error if it does not exist